source/class/qx/tool/compiler/targets/meta/Browserify.js

```diff
@@ -55,9 +55,11 @@ qx.Class.define("qx.tool.compiler.targets.meta.Browserify", {
         this.__appMeta.getEnvironmentValue("qx.compiler.applicationType") ==
         "browser"
       ) {
-        // Get a Set of unique `require`d CommonJS module names from
-        // all classes
-        for (let className in db.classInfo) {
+        // Get a Set of unique `require`d CommonJS module names from classes needed by the application
+        let classnames = this.__appMeta__P_44_0
```

Comment:
Is this what you mean? Will change with each compile.

Reply:
doh!  copied and pasted from the debugger ... 😊